src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java

```diff
@@ -46,5 +42,22 @@ public AccountManagementSteps() {
                 () -> assertThat(customerAccounts)
                         .containsExactlyInAnyOrder(firstAccount, secondAccount));
 
+        Given("^a customer wants to open an account$", () -> {
+            customer = new Customer();
+        });
```

Comment:
(Small styling comment)
Replace with lambda expression instead lambda statement. Intellij should suggest that.

Reply:
It does suggest indeed. Corrected.